### PR TITLE
Basic docs

### PR DESCRIPTION
**Pull request by tib**
- add installation guides 
- add getting started guides
- basic docs
- commands
- config example

### CODE REVIEW

File: src/content/docs/getting-started/basics/index.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+This is the home page.
+```
+
+The first part of the Markdown, located between the `---` characters, is called the front matter. This section defines additional properties for the content, such as the title, description, or the name of the template file used during the site generation process.
+
+Now that we have a simple home page, we still need a theme to render the website. For this purpose we can grab the minimal-theme from [GitHub](https://github.com/toucansites/minimal-theme/).
+
+```sh
+# Run this snippet inside the src/content folder
```
Comment (viaszkadi) [inline src/content/docs/getting-started/basics/index.md:74]: src instead of src/content

File: src/content/docs/getting-started/basics/index.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+mkdir themes
+curl -L -O https://github.com/toucansites/minimal-theme/archive/refs/heads/main.zip 
+unzip main.zip 
+mv minimal-theme-main default
```
Comment (viaszkadi) [inline src/content/docs/getting-started/basics/index.md:78]: mv minimal-theme-main themes/default, or cd themes before?

File: src/content/docs/getting-started/basics/index.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+toucan serve
+```
+
+To preview the site, navigate to [http://localhost:3000/](http://localhost:3000/) in a web browser. 
+
+The site logo might appear to be missing. Let's address this issue quickly and take a moment to explain how asset management works.
+
+## Basic assets
+
+Global website assets are stored in the `src/assets` folder. Any files placed in this folder will be recursively copied to the root directory of the final website.
```
Comment (viaszkadi) [inline src/content/docs/getting-started/basics/index.md:106]: in the `src/content/assets` folder

File: src/content/docs/getting-started/basics/index.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+
+The minimal theme uses the `logo.png` file as the main image for the website. This file must be placed inside the `images` folder. 
+
+It is also possible to create additional directories and place images, JavaScript, CSS, and other files in the `assets` folder. These files will be available as global assets when the website is built.
+
+Toucan differentiates between page assets and global assets. Each subpage can have its own assets directory, and together, the subpage and its assets are referred to as page bundles.
+
+## The 404 page
+
+Creating a custom "Not Found" page is a common use case for a website. To do this, create a special `404` directory inside the `contents` folder:
```
Comment (viaszkadi) [inline src/content/docs/getting-started/basics/index.md:126]: `content` folder

File: src/content/docs/getting-started/basics/index.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+
+[Take me home](/)
+```
+
+This page includes a "not found" image, which needs to be placed inside the `404/assets` directory. As mentioned earlier, pages can have their own assets stored within the page bundle, which, in this case, is the `404` directory.
+
+```sh
+# Inside the src/content/404 directory
+mkdir assets
+cp ~/not-found.png ./not-found.png
```
Comment (viaszkadi) [inline src/content/docs/getting-started/basics/index.md:156]: cp ~/not-found.png ./assets/not-found.png, or cd assets before

File: src/content/docs/getting-started/basics/index.md [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+
+This page is all about my website.
+```
+
+Customize the contents of the about page as desired. Regenerate the site or use the `watch` command to monitor for changes and preview the updates in the browser.
+
+
+## Site navigation
+
+The final step of this tutorial is setting up a basic menu for the website. The minimal theme supports navigation, and new menu items can be added by defining them in the site's configuration file as shown below:
```
Comment (viaszkadi) [inline src/content/docs/getting-started/basics/index.md:190]: Does this theme support navigation? I do not see the `navigation.mustache` file.

**Review by viaszkadi — CHANGES_REQUESTED** [review]

File: src/content/docs/getting-started/basics/index.md [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+This is the home page.
+```
+
+The first part of the Markdown, located between the `---` characters, is called the front matter. This section defines additional properties for the content, such as the title, description, or the name of the template file used during the site generation process.
+
+Now that we have a simple home page, we still need a theme to render the website. For this purpose we can grab the minimal-theme from [GitHub](https://github.com/toucansites/minimal-theme/).
+
+```sh
+# Run this snippet inside the src/content folder
+mkdir themes
```
Comment (viaszkadi) [inline src/content/docs/getting-started/basics/index.md:75]:
```suggestion
mkdir themes
cd themes
```

File: src/content/docs/getting-started/basics/index.md (hunk 1 above)
Comment (viaszkadi) [inline src/content/docs/getting-started/basics/index.md:74]:
```suggestion
# Run this snippet inside the src folder
```

File: src/content/docs/getting-started/basics/index.md (hunk 3 above)
Comment (viaszkadi) [inline src/content/docs/getting-started/basics/index.md:106]:
```suggestion
Global website assets are stored in the `src/content/assets` folder. Any files placed in this folder will be recursively copied to the root directory of the final website.
```

File: src/content/docs/getting-started/basics/index.md [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+This page does not exists.
+
+[Take me home](/)
+```
+
+This page includes a "not found" image, which needs to be placed inside the `404/assets` directory. As mentioned earlier, pages can have their own assets stored within the page bundle, which, in this case, is the `404` directory.
+
+```sh
+# Inside the src/content/404 directory
+mkdir assets
```
Comment (viaszkadi) [inline src/content/docs/getting-started/basics/index.md:155]:
```suggestion
mkdir assets
cd assets
```

File: src/content/docs/getting-started/basics/index.md (hunk 6 above)
Comment (tib) [reply src/content/docs/getting-started/basics/index.md:192]: yes, but it's inside the `blocks/header.mustache` file.

File: src/content/docs/getting-started/basics/index.md [hunk 9]
```diff
@@ -0,0 +1,209 @@
...
+The final step of this tutorial is setting up a basic menu for the website. The minimal theme supports navigation, and new menu items can be added by defining them in the site's configuration file as shown below:
+
+```yml
+site:
+    baseUrl: "http://localhost:3000/"
+    title: "My website"
+    navigation:
+        - label: "Home"
+          url: "/"
+        - label: "Abut"
```
Comment (viaszkadi) [inline src/content/docs/getting-started/basics/index.md:201]:
```suggestion
        - label: "About"
```

File: src/content/docs/getting-started/basics/index.md (hunk 6 above)
Comment (viaszkadi) [reply src/content/docs/getting-started/basics/index.md:192]: fixed it in the theme to make this work: https://github.com/toucansites/minimal-theme/pull/1

**Review by viaszkadi — CHANGES_REQUESTED** [review]

**Pull request merged**